### ISSUE TITLE
add term "accessibility ancestor"

### ISSUE THREAD

**Issue by pkra** [w3c/aria#2016]
Extracted from https://github.com/w3c/aria/pull/2010#discussion_r1308602573

Tooltip currently references owning element receiving focus. #2010 will update that to accessibility parent. It seems like "accessiblity ancestor" could be more fitting - but we lack that term.

**Comment by pkra** [comment]
Currently proposed in https://github.com/w3c/aria/pull/2483
